dwl.c

```diff
@@ -886,8 +886,20 @@ createnotify(struct wl_listener *listener, void *data)
 	struct wlr_xdg_surface *xdg_surface = data;
 	Client *c;
 
-	if (xdg_surface->role != WLR_XDG_SURFACE_ROLE_TOPLEVEL)
+	if (xdg_surface->role == WLR_XDG_SURFACE_ROLE_POPUP) {
+		Client *c = wlr_xdg_surface_from_wlr_surface(xdg_surface->popup->parent)->data;
```

Comment:
I think this will crash if the parent of the popup is another popup, can you check?

Reply:
Since popups are also xdg surfaces it won't crash. Although it will probably segfault later when geom.width is accessed.

Reply:
As it turns out, width and height don't have to be anything specific. Setting it to the monitor height and width seems to work